cdist/conf/type/__grub_config/manifest

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh -e
+#
+# 2017 Daniel Tschada (mail--at--moep.name)
+#
+# This file is part of cdist.
+#
+# cdist is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cdist is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with cdist. If not, see <http://www.gnu.org/licenses/>.
+#
+
+os=$(cat "$__global/explorer/os")
+os_version=$(cat "$__global/explorer/os_version")
+grub_default_path="/etc/default/grub"
+# check os
+case "$os" in
+    devuan)
+        case "$os_version" in
+            jessie) GRUB_DISTRIBUTOR=`lsb_release -i -s 2> /dev/null || echo Devuan`
```

Comment:
Lines like above should be refactored. I propose writing type's explorer that gets lsb_release -i -s from remote,
and here use it something like GRUB_DISTRIBUTOR=$(cat "$__object/explorer/lsb_release"),
without os and os_version logic - this should be handled in explorer.